django_ibmi/introspection.py

```diff
@@ -27,34 +27,51 @@
 
 from django import VERSION as djangoVersion
 
+# TODO fix pyodbc access in Introspection when doing rest of module
+# after fix to DatabaseWrapper and CursorWrapper
+
 class DatabaseIntrospection( BaseDatabaseIntrospection ):
     
     """
     This is the class where database metadata information can be generated.
     """
 
+    SQL_AUTOFIELD = -777555
+    SQL_BIGAUTOFIELD = -777444
```

Comment:
What are these for?

Reply:
I believe I pulled these from this repo https://github.com/lionheart/django-pyodbc/blob/master/django_pyodbc/introspection.py so it is most likely defining Microsoft SQL Server datatypes. For now I removed the whole reverse data types change and just put a TODO to handle that in a separate PR.

Reply:
Ok.